sumatra/versioncontrol/_mercurial.py

```diff
@@ -82,25 +78,24 @@ def has_changed(self):
 
     def diff(self):
         """Difference between working copy and repository."""
-        opts = patch.mdiff.diffopts(nodates=True)
-        diff = patch.diff(self.repository._repository, opts=opts)
-        return "".join(diff)
-
-    def content(self, hex):
-        repo = self.repository._repository
-        i = 1
-        if hex in repo.parents()[0].hex():
-            ctx = repo.parents()[0]
-            return ctx.filectx(ctx.files()[0]).data()
-        while True:
-            el = repo.parents(i)[0].hex()
-            if hex in el:
-                ctx = repo.parents(i)[0]
-                return ctx.filectx(ctx.files()[0]).data()  # presume that we have only one file [0]
-            i += 1
+        diffs = self.repository._repository.hg_diff()
+        return "".join([entry['diff'] for entry in diffs])
+
+    #def content(self, hex):
```

Comment:
I think the `content()` method was used in an earlier version of the web interface. I would like to re-enable this functionality (viewing the precise version of code used for a particular computation) in future, but it is ok to remove the method for now, we can reimplement it later.

(What is needed, I think, is the "hg cat" command; this is not at present supported by hgapi, but it doesn't look like too much work to add it).